File: T6. C# CONDITIONAL STATEMENTS AND LOOPS - EXERCISES (Java)/src/P15_NeighbourWars.java
import java.util.Scanner;

public class P15_NeighbourWars {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        int damagePesho = Integer.parseInt(in.nextLine());
        int damageGosho = Integer.parseInt(in.nextLine());

        final int HEALTH_POINTS = 10;
        int bloodPesho = 100;
        int bloodGosho = 100;
        int rounds = 1;


        while (true) {

            if (rounds % 2 != 0) {

                bloodGosho -= damagePesho;
                if (bloodGosho <= 0) {

                    System.out.printf("Pesho won in %dth round.", rounds);
                    break;
                } else {

                    System.out.printf("Pesho used Roundhouse kick and reduced Gosho to %d health.\n", bloodGosho);
                }
            } else {

                bloodPesho -= damageGosho;

                if (bloodPesho <= 0) {

                    System.out.printf("Gosho won in %dth round.", rounds);
                    break;
                } else {

                    System.out.printf("Gosho used Thunderous fist and reduced Pesho to %d health.\n", bloodPesho);
                }
            }
            if (rounds % 3 == 0) {

                bloodGosho += HEALTH_POINTS;
                bloodPesho += HEALTH_POINTS;
            }
            rounds++;
        }
    }
}

File: T6. C# CONDITIONAL STATEMENTS AND LOOPS - EXERCISES/P15.NeighbourWars.cs
using System;

namespace P15.NeighbourWars
{
    class Program
    {
        const int HEALTH_POINTS = 10;

        static void Main(string[] args)
        {
            int damagePesho = int.Parse(Console.ReadLine());
            int damageGosho = int.Parse(Console.ReadLine());

            int bloodPesho = 100;
            int bloodGosho = 100;
            int rounds = 1;


            while (true)
            {

                if (rounds % 2 != 0)
                {

                    bloodGosho -= damagePesho;

                    if (bloodGosho <= 0)
                    {

                        Console.WriteLine($"Pesho won in {rounds}th round.");
                        break;
                    }
                    else
                    {

                        Console.WriteLine($"Pesho used Roundhouse kick and reduced Gosho to {bloodGosho} health.");
                    }
                }
                else
                {

                    bloodPesho -= damageGosho;

                    if (bloodPesho <= 0)
                    {

                        Console.WriteLine($"Gosho won in {rounds}th round.");
                        break;
                    }
                    else
                    {

                        Console.WriteLine($"Gosho used Thunderous fist and reduced Pesho to {bloodPesho} health.");
                    }
                }
                if (rounds % 3 == 0)
                {

                    bloodGosho += HEALTH_POINTS;
                    bloodPesho += HEALTH_POINTS;
                }

                rounds++;
            }
        }
    }
}
